src/booking/containers/booking-home/booking-home.component.html

```diff
@@ -1,7 +1,62 @@
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <div class="hmcts-page-heading">
+        <div class="hmcts-page-heading__title">
+          <h1 class="govuk-heading-l">Work access</h1>
+        </div>
+      </div>
+      <form [formGroup]="bookingTypeForm">
+        <div class="govuk-form-group">
+          <div class="govuk-radios govuk-radios--conditional" data-module="govuk-radios">
+            <ng-container *ngIf="existingBookings && existingBookings.length > 0">
+              <div class="govuk-radios__item">
+                <input class="govuk-radios__input" id="type-0" name="bookingType" type="radio" [value]="0"
+                data-aria-controls="conditional-booking-type-0" (change)="onSelectOption(0)" formControlName="bookingType"/>
+                <label class="govuk-label govuk-radios__label" for="type-0" id="type-0-label">
+                  Choose an existing booking
+                </label>
+              </div>
+              <div id="conditional-booking-type-0" *ngIf="bookingTypeForm.get('bookingType').value === 0" class="govuk-radios__conditional">
+                <div class="govuk-form-group">
+                  <div class="govuk-grid-column-one-half" *ngFor="let booking of existingBookings" >
+                    <p>
+                      <strong>{{booking.locationName}}</strong>
+                      <span class="govuk-hint">
+                        {{booking.beginTime | date:'dd LLLL yyyy'}} to {{booking.endTime | date:'dd LLLL yyyy'}}
+                      </span>
+                    </p>
+                    <div class="govuk-button-group">
+                      <button
+                        role="button"
+                        class="govuk-button govuk-button--secondary"
+                      >
+                        Continue
+                      </button>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </ng-container>
+            <div class="govuk-radios__item">
+              <input class="govuk-radios__input" id="type-1" name="bookingType" type="radio"
+                (change)="onSelectOption(1)" [value]="1" formControlName="bookingType"/>
+              <label class="govuk-label govuk-radios__label" for="type-1">
+                Create a new booking
+              </label>
+            </div>
+            <div class="govuk-radios__item">
+              <input class="govuk-radios__input" id="type-2" name="bookingType" type="radio"
+                (change)="onSelectOption(2)" [value]="2" formControlName="bookingType"/>
+              <label class="govuk-label govuk-radios__label" for="type-2">
+                View tasks and cases
+              </label>
+            </div>
+          </div>
+        </div>  
+        <button class="govuk-button" type="button" *ngIf="newBooking.bookingOptionIndex" (click)="onEventTrigger()">Continue</button>
+      </form>
+    </div>
+
```

Comment:
Almost perfection apart from this empty line. On a serious note it would be nice if we started using the gov-ui components in the common lib, however I'll approve as it is at present.